public/app/features/alerting/unified/components/receivers/grafanaAppReceivers/onCall/onCall.ts

```diff
@@ -0,0 +1,17 @@
+import { Receiver } from '../../../../../../../plugins/datasource/alertmanager/types';
+
+export const isInOnCallIntegrations = (url: string, integrationsUrls: string[]) => {
+  return integrationsUrls.findIndex((integrationUrl) => integrationUrl === url) !== -1;
```

Comment:
Is this the same as `integrationsUrls.includes(url)`?

Reply:
💯 I don't remember why I did it this way 🤔